index.html

```diff
@@ -12,6 +12,35 @@
     ></script>
   </head>
   <body class="theme1">
+    <div id="popup-container">
+      <div id="popup">
+        <button id="popup-close">
+          <img src="./assets/icons/close.svg" alt="close-button" />
+        </button>
+        <picture class="popup-image">
+          <source
+            srcset="./assets/background/desktop-project.png"
+            media="(min-width: 768px)"
+          />
+          <img
+            class="popup-image"
+            src="./assets/background/mobile-project.png"
+            alt="project-image"
+          />
+        </picture>
+        <h2 class="popup-title text-secondary"></h2>
+        <ul id="popup-tags" class="languages"></ul>
+        <p class="popup-text"></p>
+        <div id="popup-buttons-container">
+          <button class="popup-button">
+            See Live <img src="./assets/icons/githubWhite.svg" alt="icon" />
+          </button>
+          <button class="popup-button">
+            See Source <img src="./assets/icons/see-live.svg" alt="icon" />
+          </button>
```

Comment:
- [ ] Hi @BahirHakimy, well done working on the pop-up section of your project, 👍 👍 but according to the project requirement, it is paramount that the pop-up is created dynamically and the hardcoded should be removed from the HTML file. I would kindly suggest you create the pop-up dynamically and remove the code you have in the HTML file to see if the dynamic function works fine in your JavaScript file 👍 

  - N.B: The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript.** Check an example of a student's projects: [Link 1](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) and [Link 2](https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96)